UnstoppableWallet/UnstoppableWallet/Core/Adapters/ZcashAdapter.swift

```diff
@@ -409,7 +448,7 @@ extension ZcashAdapter: IDepositAdapter {
 
     var receiveAddress: String {
         // only first account
-        synchronizer.getAddress(accountIndex: 0)
+        synchronizer.getShieldedAddress(accountIndex: 0) ?? "" //FIXME: address nullabilty issue
```

Comment:
@ant013 I'd like to discuss how to handle this kind of interface differences on _throwability_ or nullability